test/web-platform-tests/to-upstream/html/dom/dynamic-markup-insertion/document-writeln/document.writeln-03.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<title>document.writeln with multiple arguments</title>
+<link rel="author" title="Ms2ger" href="mailto:ms2ger@gmail.com">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#dom-document-writeln%28%29">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#documents-in-the-dom">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
+"use strict";
+
+test(() => {
+  const iframe = document.createElement("iframe");
```

Comment:
Nit: I have adopted the convention of placing newlines after `() => {` and before `}, "Test description");` in WPT, because the backward order makes things confusing for me otherwise. *shrug*.